source/source.go

```diff
@@ -43,3 +47,41 @@ func (s *Source) Configure(ctx context.Context, cfgRaw map[string]string) error
 
 	return nil
 }
+
+// Open initializes a Pub/Sub client.
+func (s *Source) Open(ctx context.Context, _ sdk.Position) error {
+	pubSub, err := clients.NewClient(ctx, s.cfg)
```

Comment:
Did we do ping there?

Reply:
No, there is no ping.
If the credentials are invalid, the system returns an error while creating a reference to a subscription:
https://github.com/conduitio-labs/conduit-connector-gcp-pubsub/blob/feature/MEX-157/pull-message/clients/pubsub.go#L62